paperless-ngx.json

```
@@ -0,0 +1,113 @@
+                    }
+                },
+                "environment": {
+                    "PAPERLESS_REDIS": {
+                        "description": "Set to redis://paperless-broker:6379.Redis connection URL. Do not change unless you know what you are doing.",
+                        "value": "redis://paperless-broker:6379",
+                        "label": "Redis URL"
+                    },
+                    "PAPERLESS_DBHOST": {
+                        "description": "Set to paperless-db. Hostname of the PostgreSQL database container.",
+                        "value": "paperless-db",
+                        "label": "Database Host"
+                    },
+                    "PAPERLESS_TIME_ZONE": {
+                        "description": "Enter timezone",
+                        "value": "Europe/Berlin",
+                        "label": "Timezone"
+                    }
+                }
+            },
+            "paperless-broker": {
+                "image": "docker.io/library/redis",
+                "tag": "7",
+                "launch_order": 1,
+                "volumes": {
+                    "/data": {
+                        "description": "Choose a Share for Redis data.",
+                        "label": "Redis Data Share"
+                    }
+                }
+            },
+            "paperless-db": {
+                "image": "docker.io/library/postgres",
+                "tag": "16",
+                "launch_order": 2,
+                "volumes": {
+                    "/var/lib/postgresql/data": {
+                        "description": "Choose a Share for PostgreSQL database storage.",
+                        "label": "PostgreSQL Storage"
+                    }
+                },
+                "environment": {
+                    "POSTGRES_DB": {
+                        "description": "Enter the database name for Paperless.",
+                        "label": "Database Name",
+                        "index": 1
+                    },
+                    "POSTGRES_USER": {
+                        "description": "Enter the username for the PostgreSQL database.",
+                        "label": "Database Username",
+                        "index": 2
+                    },
+                    "POSTGRES_PASSWORD": {
+                        "description": "Enter the password for the PostgreSQL database user.",
+                        "label": "Database Password",
+                        "index": 3
+                    }
+                }
+            }
+        },
+        "description": "Paperless DMS - Private & Secure document management system. <p>Based on a Docker image: <a href='https://github.com/paperless-ngx/paperless-ngx/tree/main'>GitHub Repository</a>. Available for amd64 and arm64 architectures.</p> <p> Create User like shown here (#7): <a href='https://docs.paperless-ngx.com/setup/#docker_hub'>https://docs.paperless-ngx.com/setup/#docker_hub</a> </p>",
```

```suggestion
        "description": "Paperless DMS - A private & secure Document Management System. <p>Based on the official Docker image: <a href='https://github.com/paperless-ngx/paperless-ngx/tree/main'>GitHub Repository</a>, available for amd64 and arm64 architecture.</p> <p><i>Note:</i> Create Superuser like shown here (#7): <a href='https://docs.paperless-ngx.com/setup/#docker_hub'>https://docs.paperless-ngx.com/setup/#docker_hub</a> by using the command line and entering the paperless container with <code>docker exec -it paperless /bin/bash</code></p><p>Then execute <code>python3 manage.py createsuperuser</code>, which will then prompt for a username, email address and password.</p><p>These credentials can then be used to intially login into the paperless WebUI.</p>",
```
Making the superuser creation for initial login more explicit, in addition to the link you already mentioned.